### PR TITLE
add standard_exponential to numpy frontend

### LINKED ISSUE #13418: standard_exponential

### PR DESCRIPTION
**Pull request by Aneesh02**
Closes #13418

### CODE REVIEW

File: ivy/functional/frontends/numpy/random/functions.py [hunk 1]
```diff
@@ -89,3 +89,9 @@ def shuffle(x, /):
 @from_zero_dim_arrays_to_scalar
 def standard_normal(size=None):
     return ivy.random_normal(mean=0.0, std=1.0, shape=size, dtype="float64")
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def standard_exponential(beta):
+    return ivy.gamma(1.0, beta, dtype="float64")
```
Comment (YushaArif99) [inline ivy/functional/frontends/numpy/random/functions.py:97]: are you sure the `standard exponential` function in Numpy behaves identically to the `gamma` function? Numpy provides both these functions separately so shouldn't there must be some difference?

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @Aneesh02 

Some of the tests for `test_standard_exponential` seem to be failing in the CI. You can view the detailed error log [here](https://github.com/unifyai/ivy/actions/runs/4597860171/jobs/8121118846?pr=13422)

Can you kindly fix these tests and re-request a review. Thanks! 🙂 

```

**Comment by Aneesh02** [comment]
Still working on it!

**Comment by YushaArif99** [comment]
Hey @Aneesh02 

Since there has been no activity on this PR for over a month now, I'll be closing this PR 🙂

**Pull request closed without merging**